sdk/dotnet/Config/Config.cs

```diff
@@ -33,7 +33,7 @@ public static class Config
         /// </summary>
         public static string? KeyMaterial { get; set; } = __config.Get("keyMaterial");
 
-        public static ImmutableArray<Pulumi.Docker.Config.Types.RegistryAuth> RegistryAuth { get; set; } = __config.GetObject<ImmutableArray<Pulumi.Docker.Config.Types.RegistryAuth>>("registryAuth");
+        public static Pulumi.Docker.Config.Types.RegistryAuth? RegistryAuth { get; set; } = __config.GetObject<Pulumi.Docker.Config.Types.RegistryAuth>("registryAuth");
```

Comment:
This looks breaking... do you know what happened here?

Reply:
Fixed! They added MAxItems: 1, I removed it again to ensure we don't break - it was a linting change on their side